src/nemo-file-management-properties.c

```diff
@@ -765,6 +767,7 @@ static  void
 nemo_file_management_properties_dialog_setup (GtkBuilder *builder, GtkWindow *window)
 {
 	GtkWidget *dialog;
+    GtkWidget *stack_switcher;
```

Comment:
I know. We normally use a convention of 4 spaces for indentation in new code. It displays properly in the editor but not here.